xml/System.Windows.Forms/AccessibleEvents.xml

```diff
@@ -758,6 +796,15 @@
       <MemberValue>6</MemberValue>
       <Docs>
         <summary>A shortcut menu was displayed. The system sends this event for standard menus. Servers send this event for custom menus. The system does not send the event consistently.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+Note that for a call to TrackPopupMenu(), a client will see `EVENT_SYSTEM_MENUSTART` followed almost immediately by `EVENT_SYSTEM_MENUPOPUPSTART` for the popup being shown.
+
+          ]]></format>
+        </remarks>
```

Comment:
Same. Feel free to apply the suggestion if this is not something that matters to people reading the API description in MS Docs:

```suggestion
        <remarks>To be added.</remarks>
```